demos/continuous_batching/rag/README.md

```diff
@@ -36,8 +36,46 @@ ovms.exe --add_to_config models --model_name OpenVINO/bge-reranker-base-fp16-ov
 :::
 ::::
 
+### 2. Download the preconfigured models using ovms --pull option for models outside [HugginFaces Hub OpenVINO organization](https://huggingface.co/OpenVINO) in HuggingFace Hub.
```

Comment:
I propose to start adding this option in all demos after we publish the image with optimum-cli.